### PR TITLE
score, level and container styling

### LINKED ISSUE #30: Score Display
Need to create and style the component that display's the current score. The data is already available in the store. See #26 for more details.

### PR DESCRIPTION
**Pull request by MattGiesa**
closes #30 

adds the score component,
moves the side bar contents around,
adds styling to board container and next-pieces container,
adds level to top of board container

Vue:

image

Angular:

image

React:

image

### CODE REVIEW

File: src/front-end/vue/components/board.ts [hunk 1]
```diff
@@ -13,13 +13,20 @@ export const Board = () => ({
...
+    },
     styles: Object,
   },
   template: `
-    <div class="${ board }" v-bind:style="styles">
-      <div class="${ flexGrowRow }" v-for="(row, i) in board" v-bind:key="i">
-        <bd-tile v-for="(tile, j) in row" v-bind:value="tile" v-bind:key="j">
-        </bd-tile>
+    <div class="ba bw2 b--vue-green mr4 shadow-vue-green">
+      <h4 class="black bg-vue-green f3 mb0 mt0 tc">LEVEL {{level}}</h4>
```
Comment (MattGiesa) [inline src/front-end/vue/components/board.ts]: I added the wrapper <div> and the <h4>, the rest are white space changes

File: src/front-end/vue/components/next-pieces.ts [hunk 2]
```diff
@@ -12,14 +11,15 @@ export const NextPieces = () => ({
...
-  <div class="${flexGrowShrink} ${flexCol}">
-    <h3>Next:</h3>
-    <bd-block 
-      v-for="(block, i) in preview" 
-      v-bind:block="block"
-      v-bind:key="i"
-    >
-    </bd-block>
+  <div class="ba bw2 b--vue-green mb4 shadow-vue-green">
+    <h4 class="black bg-vue-green f3 mb0 mt0 tc">NEXT</h4>
```
Comment (MattGiesa) [inline src/front-end/vue/components/next-pieces.ts]: I just noticed this was previously a `<h3>`. I didn't change it to h4 for any particular reason, I can change it back

**Comment by MattGiesa** [comment]
should I update all frameworks in one PR?

**Comment by bennett000** [comment]
All frameworks in one PR is great, thanks! @MattGiesa

**Comment by MattGiesa** [comment]
all frameworks have been updated

**Pull request merged**